target/linux/mpc85xx/base-files/etc/board.d/02_network

```diff
@@ -32,6 +32,14 @@ tplink,tl-wdr4900-v1)
 watchguard,firebox-t10)
 	ucidef_set_interfaces_lan_wan "eth1 eth2" "eth0"
 	;;
+watchguard,t30-w)
+	ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"
+	ucidef_set_interface_macaddr "wan" "$(mtd_get_mac_text nor-cfg0 0x1830)"
```

Comment:
This should be handled in dts with Mac-base

Reply:
Thanks, updated